docker/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM quinoacomputing/buildenv:gentoo
```

Comment:
Can you rename this file to differentiate better from the other docker files in that directory? E.g., `Dockerfile.quinoa-build-travis` or something like that.